external/elasticsearch/src/main/java/com/digitalpebble/stormcrawler/elasticsearch/persistence/AbstractSpout.java

```diff
@@ -164,13 +164,19 @@ public void open(
     protected final boolean addHitToBuffer(SearchHit hit) {
         Map<String, Object> keyValues = hit.getSourceAsMap();
         String url = (String) keyValues.get("url");
+
+        Map<String, List<String>> metadatas = (Map<String, List<String>>) keyValues.get("metadata");
```

Comment:
what happens to the metadatas object afterwards? it would feel more logical if it did something to the actual metadata object which is being returned.